bentoml/_internal/server/cli/api_server.py

```diff
@@ -42,6 +42,11 @@
     type=click.Path(exists=True),
     help="Working directory for the API server",
 )
+@click.option(
```

Comment:
this will require changes on Yatai deployment operator side as well, right? @yetone @bojiang

Reply:
looks like Yatai does not run api_server workers from here?

I guess when we add metrics to runner_app, we will need to update the runner_app CLI command to include a prometheus directory?

Reply:
No. This option is not required. On the other hand, the e2e test of simulated yatai passed so no worry. In Yatai the default directory will always work, so no need to specify it.